components/servo/lib.rs

```diff
@@ -979,6 +979,14 @@ impl Servo {
                         .show_form_control(webview, FormControl::SelectElement(prompt));
                 }
             },
+            EmbedderMsg::ShowColorPicker(webview_id, current_color, position, ipc_sender) => {
+                if let Some(webview) = self.get_webview_handle(webview_id) {
+                    let prompt = ColorPicker::new(current_color, position, ipc_sender);
+                    webview
+                        .delegate()
+                        .show_form_control(webview, FormControl::ColorPicker(prompt));
+                }
+            },
         }
```

Comment:
If it's not too much of a hassle, would you mind combining the `EmbedderMsg::ShowColorPicker` and EmbedderMsg::ShowSelectElementMenu` into a single `EmbedderMsg::ShowFormControl` message? I think that having the `EmbedderMsg` mirror the delegate messages is conceptually nicer.